main.nf

```diff
@@ -2932,7 +2932,7 @@ process print_nuclear_contamination{
     params.run_nuclear_contamination
 
     input:
-    val 'Contam' from ch_from_nuclear_contamination.map { it[7] }.collect()
+    path 'Contam' from ch_from_nuclear_contamination.map { it[7] }.collect()
```

Comment:
```suggestion
    path Contam from ch_from_nuclear_contamination.map { it[7] }.collect()
```

Reply:
with the quotes, the input files are staged as "Contam1", "Contam2",[...]. This breaks the sample names in the output file. 
The omission of the quotes will keep the original file names, and correctly use the Library_IDs in the output file.

Reply:
Thanks!
I was wondering why the files came back named that way...